swift/Samples/SampleSwiftUIApp/SampleSwiftUIApp.xcodeproj/project.pbxproj

```diff
@@ -0,0 +1,366 @@
+// !$*UTF8*$!
```

Comment:
Rather than checking in an xcodeproj, would using `be pod gen` work here?

Reply:
Yes! I've created a new issue to track this: https://github.com/square/workflow/issues/930

This one is similar to the other sample apps in the repo.